draft-irtf-cfrg-frost.md

```diff
@@ -684,10 +690,11 @@ the network channel is additionally authenticated; confidentiality is not requir
 ## Round One - Commitment {#frost-round-one}
 
 Round one involves each signer generating nonces and their corresponding public commitments.
-A nonce is a pair of Scalar values, and a commitment is a pair of Element values.
-
-Each signer in round one generates a nonce `nonce = (hiding_nonce, binding_nonce)` and commitment
-`comm = (hiding_nonce_commitment, binding_nonce_commitment)`.
+A nonce is a pair of Scalar values, and a commitment is a pair of Element values. Each signer's
```

Comment:
The first line says "involves each signer generating nonces", further lines say "a nonce is a pair". I still always think of two nonces, one hiding and one binding, and their respective commitments, for each signer, for each signature

// @chelseakomlo

Reply:
I'm going to merge without a change here now. @chelseakomlo can submit a PR to address on top if she thinks it should change.

Reply:
ok